### PR TITLE
Import Kitematic documentation into docs.docker.com

### PR DESCRIPTION
**Pull request by SvenDowideit**
replaces #481

not ready for merge.

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@jeffdm @mchiang0610 can you please check the userguide.md for factual errors and missing bits?

@moxiegirl @fredlf can you please poke holes in my engrish, I've been looking at this too long.

**Comment by jeffdm** [comment]
Looking

**Comment by SvenDowideit** [comment]
damnit, wrong button.

OH, and I'm wondering if we should remove the Known issues and FAQ, and instead link to them as files in this repository (that way they can be updated without needing re-publishing too)

File: docs/userguide.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+The "New Container" page lets you search for and select from images on the Docker Hub.
+When you've found the image you want to run, you can click "Create" to pull, create,
+and run the container.
+
+![Nginx create](../assets/browse-images.png)
+
+## Working with a container
+
+If you select a non-running container, either stopped, or paused, you will be able to
+"Restart" or "Stop" the container using the icons. You can also view the entire PID-0
```
Comment (jeffdm) [inline docs/userguide.md]: PID-0 might be a little advanced for some users - do we refer to this as anything else? The `main process` in the container?

File: docs/userguide.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+"Restart" this container.
+
+By selecting a running container from the left list, you can see some state information
+for your container - either a preview of the HTML output for a container that has a web
+server, the PID-0 logs, and any container volumes that have been configured.
+
+![Redis container in Kitematic](../assets/cli-redis-container.png)
+
+The summary page will show different things depending on the image metadata. If
+port 80 is `EXPOSED`, then Kitematic assumes its a web page, and will show a preview
```
Comment (jeffdm) [inline docs/userguide.md]: Ports included are ['80', '8000', '8080', '3000', '5000', '2368', '9200', '8983'] but we may add more.

File: docs/userguide.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+You can then scroll through the logs from the current running container. Note that
+if you make changes to the container Settings, then the container will be restarted,
+so will reset this log view.
+
+### Starting a terminal in a container
+
+The "Terminal" icon at the top of the container summary will `docker exec sh <your container>`.
+This will allow you to make quick changes, or to debug a problem. 
+
+> **Note**: Your execed `sh` process will not have the same environment settings
```
Comment (jeffdm) [inline docs/userguide.md]: Great add!

**Comment by jeffdm** [comment]
Wow, Sven, `userguide.md` is awesome! It's going to answer so many questions we get from users on a regular basis.

FAQ shouldn't change too much, so that one can be moved. Known issues we can keep as a file in this repo.

Awesome stuff! Once @moxiegirl @fredlf  have a look let's get this merged!

File: docs/userguide.md (hunk 1 above)
Comment (SvenDowideit) [inline docs/userguide.md]: yeah - and put PID-0 in braces and perhaps link to somewhere. yup

File: docs/userguide.md (hunk 2 above)
Comment (SvenDowideit) [inline docs/userguide.md]: like... a cfg settings... :p

File: docs/rethinkdb-dev-database.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+list as shown above. This will download & run a RethinkDB container within a few
+minutes. Once it's done, you'll have a local RethinkDB database up and running.
+
+![Rethink container](../assets/rethink-container.png)
+
+Let's start using it to develop a node.js app. For now, let's figure out which
+IP address and port RethinkDB is listening on. To find out, click the `Settings`
+tab and then the `Ports` section:
+
+![Rethink create button](../assets/rethink-create.png)
```
Comment (SvenDowideit) [inline docs/rethinkdb-dev-database.md]: this is the wrong image.

File: docs/rethinkdb-dev-database.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+You can see there that for RethinkDB port `28015`, the container is listening on
+host `192.168.99.100` and port `49154` (in this example - ports may be different
+for you). This means you can now reach RethinkDB via a client driver at
+`192.168.99.100:49154`. Again, this IP address may be different for you.
+
+### (Advanced) Saving Data into RethinkDB with a local Node.js App
```
Comment (SvenDowideit) [inline docs/rethinkdb-dev-database.md]: explain that `local` means running on OSX, and that the rethinkdb is essentially `remote`

File: docs/nginx-web-server.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+In this example website we'll be serving the popular 2048 game, as shown below.
+Let's get to it!
+
+![2048 game](../assets/nginx-2048.png)
+
+#### Running the Nginx Web Server Container
+
+First, if you haven't yet done so, [download and start
+Kitematic](./index.md). Once installed and running, the app
```
Comment (SvenDowideit) [inline docs/nginx-web-server.md]: mmm, rephrase this `the app` feels odd to me.

File: docs/nginx-web-server.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+![download Nginx hello world](../assets/nginx-hello-world.png)
+
+Once it's done downloading you should see a quick preview of the example website
+that comes with the container, as shown below. Click on the preview to see the
+result in your own browser.
+
+![Nginx preview](../assets/nginx-preview.png)
+
+**What just happened?** Kitematic downloaded the `kitematic/hello-world-nginx`
```
Comment (SvenDowideit) [inline docs/nginx-web-server.md]: downloaded (also know as pulled)

**Comment by fredlf** [comment]
Agree with the comments, let's address them in follow on PR's. LGTM

**Comment by jeffdm** [comment]
LGTM, merging.

**Pull request merged**